### PR TITLE
Create app

### PR DESCRIPTION
**Pull request by AndresMillsGallego**

### CODE REVIEW

**Review by L-nobilis — APPROVED** [review]

File: src/Components/ResultsDisplay.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+          </Accordion>
+          <Accordion id="restaurantResults" className="w-50 m-3 p-2">
+            {/* these will be dynamically generated using array.map */}
+            <Accordion.Item key="i">
+              <Accordion.Header>
+                Title Text Goes here
+              </Accordion.Header>
+              <Accordion.Body>
+                Body content goes here
+                <ul>
```
Comment (44thm0820) [inline src/Components/ResultsDisplay.js:50]: Maybe add react-bootstrap component list instead of html list elements?

File: .env-sample [hunk 2]
```diff
@@ -0,0 +1 @@
+REACT_APP_SERVER=http://localhost:3001
```
Comment (44thm0820) [inline .env-sample:1]: add all other variable names and put placeholders <INSERT>...

**Review by 44thm0820 — CHANGES_REQUESTED** [review]

File: src/Components/PinballForm.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { Form, Button, FormGroup } from 'react-bootstrap';
+import '../pinballform.css'
+
+class PinballForm extends React.Component {
+
+
+  handleSubmit = (e) => {
```
Comment (dbrian57) [inline src/Components/PinballForm.js:8]: I would suggest `event` instead of `e` for readability purposes.

**Review by dbrian57 — APPROVED** [review]
One suggestion inline, but otherwise :shipit:

**Pull request merged**